tutorials/hello-kubernetes/README.md

```diff
@@ -53,158 +53,29 @@ Expected output in a fresh Kubernetes cluster without Dapr installed:
-
-✅  Deploying the Dapr control plane to your cluster...
-✅  Success! Dapr has been installed to namespace dapr-system. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
-```
-
-> Without the `--wait` flag the Dapr CLI will exit as soon as the kubernetes deployments are created. Kubernetes deployments are asyncronous by default, so we use `--wait` here to make sure the dapr control plane is completely deployed and running before continuing.
-
-<!-- Introducing sleep here to first let the previous execution complete -->
 <!-- STEP
 name: Check dapr status
-tags:
-  - normal-run
 -->
 
 ```bash
 dapr status -k
 ```
-
 <!-- END_STEP -->
 
 You will see output like the following. All services should show `True` in the HEALTHY column and `Running` in the STATUS column before you continue.
 
 ```
   NAME                   NAMESPACE    HEALTHY  STATUS   REPLICAS  VERSION  AGE  CREATED
-  dapr-operator          dapr-system  True     Running  1         1.0.1    13s  2021-03-08 11:00.21
-  dapr-placement-server  dapr-system  True     Running  1         1.0.1    13s  2021-03-08 11:00.21
-  dapr-dashboard         dapr-system  True     Running  1         0.6.0    13s  2021-03-08 11:00.21
-  dapr-sentry            dapr-system  True     Running  1         1.0.1    13s  2021-03-08 11:00.21
-  dapr-sidecar-injector  dapr-system  True     Running  1         1.0.1    13s  2021-03-08 11:00.21
-```
-
-### Step 2 - Create and configure a state store
-
-Dapr can use a number of different state stores (Redis, CosmosDB, DynamoDB, Cassandra, etc) to persist and retrieve state. This demo will use Redis.
-
-1. Follow [these steps](https://docs.dapr.io/getting-started/tutorials/configure-state-pubsub/#step-1-create-a-redis-store) to create a Redis store.
-2. Once your store is created, add the keys to the `redis.yaml` file in the `deploy` directory.
-   > **Note:** the `redis.yaml` file provided in this quickstart will work securely out-of-the-box with a Redis installed with `helm install bitnami/redis`. If you have your own Redis setup, replace the `redisHost` value with your own Redis master address, and the redisPassword with your own Secret. You can learn more [here](https://docs.dapr.io/operations/components/component-secrets/).
-3. Apply the `redis.yaml` file and observe that your state store was successfully configured!
-
-<!-- As part of previous dev-init, expecting that the state store component is already there for hello-kubernetes run. At the end of this run, it will be deleted.-->
-<!-- STEP
-name: Deploy redis config
-sleep: 1
-expected_stdout_lines:
-  - "component.dapr.io/statestore configured"
-tags:
-  - normal-run
--->
-
-```bash
-kubectl apply -f ./deploy/redis.yaml
-```
-
-<!-- END_STEP -->
-
-```bash
-component.dapr.io/statestore created
+  dapr-dashboard         dapr-system  True     Running  1         0.14.0   41s  2024-07-22 13:53.05  
```

Comment:
This output is now different in v1.14 is this matters. See https://github.com/dapr/cli/pull/1434
Need to add the scheduler service